src/Symfony/Component/Translation/Translator.php

```diff
@@ -290,11 +290,11 @@ protected function getLoaders()
      *
      * @return array[array] indexed by catalog
      *
-     * @deprecated since version 2.8, to be removed in 3.0. Use TranslatorBagInterface::getCatalogue() method instead.
+     * @deprecated since version 2.8, to be removed in 3.0. Rely on TranslatorBagInterface::getCatalogue() method instead.
```

Comment:
We always use `Use ... instead` in deprecation messages. I don't think this change is worth.